PARAMETERS_MAPPING/parameters.csv

```diff
@@ -196,4 +196,17 @@ unique_id;cf_standard_name;cf_alias_name;long_name;imos_vocabulary_name;imos_voc
 201;;;hail_intensity_10min_sample;;;
 202;thickness_of_rainfall_amount;;rainfall_amount_since_9am;;;
 203;;;rainfall duration;;;
-204;rainfall_rate;;rainfall intensity_10min_sample;;;
\ No newline at end of file
+204;rainfall_rate;;rainfall intensity_10min_sample;;;
+850;number_concentration_of_biological_taxon_in_sea_water;;total abundance;Abundance of biota;;
+851;;;cell volume;Biovolume;;Mean size of cell
+852;;;length;;;Average total length
+853;mass_concentration_of_zooplankton_expressed_as_carbon_in_sea_water;;total zooplankton carbon;;;
+854;;;Shannon evenness;;;
+855;;;ratio;;;
```

Comment:
Again, this needs to be more specific. Ratio of what to what?
Looks like there are a couple of different ratio columns, which will need different parameter names.

Reply:
I was expecting there would be some other column to add more specific comments. I had to ask you this indeed

Reply:
Just put it all in the long name. That should be a full description of the parameter.

Reply:
These are fine, I added two different ratios